app/api/v1/veiws.py

```diff
@@ -0,0 +1,21 @@
+from app.api.v1 import  ParcelModel
+from flask_restful import Resource
+from flask import response,jsonify,request
+
+ class DataParcel(Resource)
```

Comment:
Unexpected indentation of this class.  I would recommend installing a linter plugin in your editor to catch such edge cases.